### PR TITLE
Update index.html with items from issues and meeting

### PR DESCRIPTION
**Pull request by mjkoster**
Issues #34 and #36 and items from the March 13th meeting review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,7 +142,7 @@ <h3>Protocol Methods and Options</h3>
             the message type, the semantic intention of the message. As a starting
             point, a superset of REST and PubSub can cover most standard
             communication patterns. Common methods are GET, PUT, POST, DELETE,
-            PATCH, PUBLISH, and SUBSCRIBE.
+            OBSERVE, PUBLISH, and SUBSCRIBE.
```
Comment (mkovatsc) [inline index.html]: OBSERVE is not a method. PATCH was.
If PATCH was not common enough, then best remove both.

File: index.html [hunk 2]
```diff
@@ -127,6 +127,13 @@ <h2>Protocol Adaptation</h2>
           (e.g., HTTP, CoAP, MQTT, etc.) is identified, the following adaptions
           specifiy the particular use within the given Platform.
         </p>
+        <p class="ednote" title="The W3">
```
Comment (mkovatsc) [inline index.html]: "The W3"?

File: index.html [hunk 3]
```diff
@@ -1045,7 +1041,7 @@ <h1>Examples of Thing Descriptions including protocol bindings</h1>
               "form": [
                 {
                   "href": "/example/light/currentdimmer",
-                  "rel": ["readData", "writeData"],
+                  "rel": ["readProperty", "writeData"],
```
Comment (mkovatsc) [inline index.html]: writeData still needs fixing

**Comment by mkovatsc** [comment]
I was held up and need to do my pass tomorrow morning.
If I shall fix as proposed in my comments, simply do not add any more commits I will merge and fix after.
If you have the time, just add commits to the PR ;)

**Comment by mjkoster** [comment]
I fixes these and a couple more I found along the way.

Added commits to the open PR

Best regards,

Michael

**Comment by mkovatsc** [comment]
Thanks!

**Pull request merged**